layers/synchronization2.cpp

```diff
@@ -397,6 +401,10 @@ DeviceFeatures::DeviceFeatures(uint32_t api_version, const VkDeviceCreateInfo* c
     if (api_version >= VK_MAKE_VERSION(1, 2, 0)) {
         timelineSemaphore = true;
     }
+    if (create_info->pEnabledFeatures != nullptr) {
```

Comment:
Is this guaranteed to be nullptr if there's a GPDP2 structure in pNext?

Reply:
VUID-VkDeviceCreateInfo-pNext-00373 
If the <code>pNext</code> chain includes a <a href=\"#VkPhysicalDeviceFeatures2\">VkPhysicalDeviceFeatures2</a> structure, then <code>pEnabled  Features</code> must be <code>NULL</code>